hyperjumptech/monika
# feature(config): add HAR file config definition

# Monika PR
Fix #274 
## What issue this PR is trying to solve
Currently the configuration is only based on monika.json specific config. Need to add capability to read configuration from HAR file.
## How this PR solve the issue
Add capability to run probe read from HAR file definition.
## How to test
1. run monika with `--har` flag set with the path to the HAR file 
2. It should be running well to monitor url set from HAR file
